sql/parse/parse.go

```diff
@@ -273,6 +273,16 @@ func convert(ctx *sql.Context, stmt sqlparser.Statement, query string) (sql.Node
 		return plan.NewShowPrivileges(), nil
 	case *sqlparser.Flush:
 		return convertFlush(ctx, n)
+	case *sqlparser.Prepare:
+		newStmt, err := sqlparser.Parse(n.Expr)
+		if err != nil {
+			return nil, err
+		}
+		return convertPrepare(ctx, newStmt, n)
```

Comment:
(minor) lines 277-280 feel pretty coupled with `convertPrepare`, so you might try pushing the responsibility for parsing the inner statement into the `convertPrepare` function and keep this large convert function a little simpler.